dbms/src/Common/TiFlashMetrics.h

```diff
@@ -1247,6 +1248,8 @@ namespace tests
 struct TiFlashMetricsHelper;
 }
 
+using KeyspaceID = UInt32;
```

Comment:
We can remove the following line 1302 that duplicate this declaration `using KeyspaceID = UInt32;`